packages/sdks/snippets/sveltekit/src/routes/landing-page/NavBar.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+  import type { BuilderContent } from '@builder.io/sdk-svelte';
+  export let links: BuilderContent | null;
```

Comment:
you're typing links as `BuilderContent | null` but it will only get passed to this component when links is defined because of `{#if data.links} <NavBar {data.links} /> {/if}`